services/app-web/src/components/SubmissionSummarySection/ContractDetailsSummarySection/ContractDetailsSummarySection.tsx

```diff
@@ -97,16 +113,29 @@ export const ContractDetailsSummarySection = ({
         submission,
         contractSupportingDocuments,
         submissionName,
+        isPreviousSubmission,
     ])
+
+    const renderDownloadButton = () => {
```

Comment:
pull this out into its own function, please. Anonymous functions that render JSX are how we got a lot of our Cypress flakes b/c the component is unmounted and remounted on every render.